fapolicy_analyzer/ui/actions.py

```diff
@@ -41,6 +42,24 @@
 RECEIVED_GROUPS = "RECEIVED_GROUPS"
 ERROR_GROUPS = "ERROR_GROUPS"
 
+INIT_DAEMON = "INIT_DAEMON"
+DAEMON_INITIALIZED = "DAEMON_INITIALIZED"
+
+REQUEST_DAEMON_START = "REQUEST_DAEMON_START"
+RECEIVED_DAEMON_START = "RECEIVED_DAEMON_START"
+ERROR_DAEMON_START = "ERROR_DAEMON_START"
+
+REQUEST_DAEMON_STOP = "REQUEST_DAEMON_STOP"
+RECEIVED_DAEMON_STOP = "RECEIVED_DAEMON_STOP"
+ERROR_DAEMON_STOP = "ERROR_DAEMON_STOP"
+
+RECEIVED_DAEMON_STATUS_UPDATE = "RECEIVED_DAEMON_STATUS_UPDATE"
+ERROR_DAEMON_STATUS_UPDATE = "ERROR_DAEMON_STATUS_UPDATE"
+
+REQUEST_DAEMON_RELOAD = "REQUEST_DAEMON_RELOAD"
+RECEIVED_DAEMON_RELOAD = "RECEIVED_DAEMON_RELOAD"
+ERROR_DAEMON_RELOAD = "ERROR_DAEMON_RELOAD"
```

Comment:
It doesn't look like any of your `ERROR` actions are ever handled by a reducer that would update the state with the `error` value.  Have you tested error handling?

Reply:
They were there for consistency with the other actions. Right now there is no error handling.

Reply:
Ok, I'd like to see error handling added before merging this.

Reply:
Error state actions are generated in the exception block in the daemon_feature.py code. I did not add associated reducer code because fapd return errors/exceptions are still TBD and did not know what the next state of the app should be, i.e. what do we want to do if Start or Stop fail? Notification pop-up? or termination? There are associated event dispatches with the errors but no changes to the app state occur.